cluster-autoscaler/hack/update-mods.sh

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Kubernetes Authors.
+#
```

Comment:
General comment on calling go mod.

Please redirect stderr to `/dev/null`. It clutters the output.
Or better than using `/dev/null`  create some debug log and pass stderr there.
You can take logging bits from update-vendor.sh

Reply:
Done